raft_learner_recovery_rate doesn't require restart (#907)
redpanda-data/documentation: modules/reference/pages/properties/object-storage-properties.adoc

=== cloud_storage_bucket

> AWS or GCP bucket that should be used to store data.

Reviewer: No Azure?

Reviewer 2: no, Azure uses a different property

> AWS or GCP bucket or container that should be used to store data.

Reviewer 2: ```suggestion
AWS or GCP bucket that should be used to store data.
```

> WARNING: To minimize the risk of data loss do not modify this property after writing data to a cloud bucket. 

Reviewer 2: By saying "to minimize the risk of data loss," it sounds a bit like there's still some inherent data loss, you can just minimize it. Could we say, "WARNING: Modifying this property after writing data to a bucket could cause data loss."

Reviewer 2: Should we also add a warning on the [TS page](https://deploy-preview-907--redpanda-docs-preview.netlify.app/current/manage/tiered-storage/)?

Reviewer 3: This property is mentioned in 5 pages
remote-read-replicas
tiered-storage
k-crd
object-storage-properties (here)
troubleshoot/inspect-bundle

I thought about adding this warning to all these pages but it feels redudant. But we can make it a partial and add to all these pages. WDYT?

Reviewer 2: The Tiered Storage & CRD pages link to this reference page with the warning, so that's probably fine. I think the RRR page should either add links to the properties (if you have time) or include the warning.

> WARNING: To minimize the risk of data loss do not modify this property after writing data to a cloud bucket. 

Reviewer 3: ```suggestion
WARNING: Modifying this property after writing data to a bucket could cause data loss.
```
